### ISSUE TITLE
Having to open each account twice

### ISSUE THREAD

**Issue by nickolasd64** [Healix/Gw2Launcher#194]
I have to open each account, close the launcher, then open it again for the auto username and password to actually start putting them in

**Comment by Healix** [comment]
Use the [beta version](https://github.com/Healix/Gw2Launcher/raw/master/Gw2Launcher/bin64/Beta/Gw2Launcher.exe).

**Comment by konradpawlak** [comment]
Yes! It works for me now thanks you! Also PSA -> you dont have to add addresses again. all alts are already added to the beta!

**Comment by alexchiriac** [comment]
Thanks, @Healix!
